js/modal/jquery.modalWindow.js

```diff
@@ -0,0 +1,102 @@
+/** @format */
+
+(function ($) {
+  const successType = {
+    text: "modal-text__success",
+    containet: "modal-containet__success",
+    button: "modal-btn_success",
+  };
+  const errorType = {
+    text: "modal-text__error",
+    containet: "modal-container__error",
+    button: "modal-btn_error",
+  };
+  const confirmType = {
+    text: "modal-text__confirm",
+    containet: "modal-container__confirm",
+    button: "modal-btn_confirm",
+  };
+
+  const defauls = {
+    content: "success",
+    buttonText: ["OK"],
+    type: successType,
+  };
+
+  class Modal {
+    constructor(element, options) {
+      this.config = $.extend({}, defauls, options);
+      this.element = element;
+      this.init();
+    }
+
+    toggleModal() {
+      $("div.modal").toggleClass("is-hidden");
+      $("div.backdrop").toggleClass("backdrop");
+      $("body").toggleClass("modal-open");
```

Comment:
"is-hidden"/"backdrop"/"modal-open" - this strings could be moved to class instance variable